ui/components/app/wallet-overview/eth-overview.test.js

```diff
@@ -259,14 +258,16 @@ describe('EthOverview', () => {
       };
       const mockedStore = configureMockStore([thunk])(mockedAvalancheStore);
 
-      const { queryByTestId, queryByText } = renderWithProvider(
+      const { queryByTestId, debug } = renderWithProvider(
         <EthOverview />,
         mockedStore,
       );
+      // Log the entire component structure
+      debug();
       const bridgeButton = queryByTestId(ETH_OVERVIEW_BRIDGE);
       expect(bridgeButton).toBeInTheDocument();
       expect(bridgeButton).toBeEnabled();
-      expect(queryByText('Bridge').parentElement).not.toHaveAttribute(
+      expect(bridgeButton.parentElement).not.toHaveAttribute(
```

Comment:
The ButtonBase has more elements so we are looking for the parent of the button component which will be the tooltip